content/docs/Other guides/configuring-limits.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Configuring limits"
+linkTitle: "Configuring limits"
+weight: 4
```

Comment:
Maybe update the weight to '2', so that this sits above the troubleshooting guides? Don't worry if this is tricky, I can update it after we merge your pull request.